toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -118,12 +118,14 @@ else
 fi
 
 # ============ Populate SRPMS ============
-# Populate ${repo_path}/build/INTERMEDIATE_SRPMS with SRPMs, that can be used to build RPMs in the container
-pushd "${repo_path}/toolkit"
-echo "Populating Intermediate SRPMs..."
-if [[ ( ! -f "${repo_path}/toolkit/out/tools/srpmpacker" ) ]]; then build_tools; fi
-make input-srpms SRPM_FILE_SIGNATURE_HANDLING="update" > /dev/null
-popd
+# Populate ${repo_path}/build/INTERMEDIATE_SRPMS with SRPMs, that can be used to build RPMs in the container (only required in build mode)
+if [[ "${mode}" == "build" ]]; then
+    pushd "${repo_path}/toolkit"
+    echo "Populating Intermediate SRPMs..."
+    if [[ ( ! -f "${repo_path}/toolkit/out/tools/srpmpacker" ) ]]; then build_tools; fi
+    make input-srpms SRPM_FILE_SIGNATURE_HANDLING="update" > /dev/null
```

Comment:
Question: do we build all `SRPM`s or do we honor the `SRPM_PACK_LIST` ?
Just incase anyone wants to only build a single srpm.

Reply:
SRPM_PACK_LIST works independently. If you don't specify that, it builds all SRPMs